### ISSUE TITLE
[Task] Fix Keyboard A11y of links inside Doc Status Alerts

### ISSUE THREAD

**Issue by bacitracin** [department-of-veterans-affairs/va.gov-team#127648]
### Feature

document-status

### Team

bmt-team-2 (Bee’s Knees)

### Issue Type

Task

### Summary

Keyboard navigation of links within the alerts is broken

### Description / Context

Noticed this while working on the accessibility testing artifact for Document Status. There are problems with keyboard accessibility of the contents within the "Document submission started on XX" green success alert and the "We need you to submit files by mail or in person" red error alert.

See the video, which has the Keycastr app on so you can see what I'm clicking on tab and navigating around.

Basically 2 problems.
1. When you initially submit a file and get the green alert: You can TAB one time to the "Check the Status of your submission" link, then TAB again to the "X" close button in the top right corner of the green alert. The focus order is incorrect. You should be able to TAB to the X first, then the link (focus order should be logical, so in this case top to bottom, left to right).
2. TAB out of the alert. Then TAB +SHIFT to try to go back to the link & the X button. It's not possible. You can't land there anymore.

The red "We need you to submit files by mail or in person" doesn't have an X close button, but it does have problem #2, where you can't return to the link via keyboard once you've TAB-bed out of the alert contents.

https://github.com/user-attachments/assets/dbf1b8e0-f8dd-4d6e-bb81-daea9129f91d

### Risks / Dependencies

- Risk: Problem 1 is annoying, but problem 2 is a potential launch blocker at staging review
